src/DynamoRevit/ViewModel/RevitWatch3DViewModel.cs

```diff
@@ -374,14 +374,14 @@ private void DeleteKeeperElement()
    
             // Never access the current document with an invalid keeperId
             // See comment at the beginning of this method.
-            var dbDoc = DocumentManager.Instance.CurrentDBDocument;
+            var dbDoc = DocumentManager.Instance.CurrentUIApplication.ActiveUIDocument.Document;
```

Comment:
I suppose that `DocumentManager.Instance.CurrentDBDocument` and `DocumentManager.Instance.CurrentUIDocument.Document` are the same, right?

Reply:
DocumentManager.Instance.CurrentDBDocument and DocumentManager.Instance.CurrentUIDocument would be null here, when shutdown. Only DocumentManager.Instance.CurrentUIApplication can remain valuable.